src/type_exports.ts

```diff
@@ -60,6 +60,7 @@
  * option TypeScript will not emit declarations for this code.
  */
 
-export { CallAPIOptions, ClusterClient } from './core/server/elasticsearch';
-export { Logger, LoggerFactory } from './core/server/logging';
-export { PluginInitializerContext, PluginName, PluginStartContext } from './core/server/plugins';
+import * as Public from './core/public';
+import * as Server from './core/server';
+
+export { Server, Public };
```

Comment:
Can we just kill type exports now and have kibana.d.ts do these imports? We discussed this in the weekly (you were at X-School) and decided that the annoyances of modifying type_exports doesn't make sense anymore. Seems like this would be a good PR to kill it in.

Reply:
Sounds like a good idea :+1: